<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallpaperWidget</class>
 <widget class="QWidget" name="WallpaperWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="switch_groupbox">
     <property name="title">
      <string>功能开关</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="on_radio">
        <property name="toolTip">
         <string>启用壁纸插件</string>
        </property>
        <property name="text">
         <string>开启</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="off_radio">
        <property name="toolTip">
         <string>关闭壁纸插件</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="function_groupbox">
     <property name="title">
      <string>功能设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="once_radio">
        <property name="text">
         <string>仅播放当前壁纸</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="loop_radio">
        <property name="text">
         <string>列表循环播放</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="random_radio">
        <property name="text">
         <string>随机播放</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="play_groupbox">
     <property name="title">
      <string>播放设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>音量(仅视频可用)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="volume_slider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="mute_checkbox">
          <property name="text">
           <string>静音</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="prev_button">
          <property name="text">
           <string>上一张壁纸</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fast_backward_button">
          <property name="text">
           <string>快退</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="play_pause_button">
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="fast_forward_button">
          <property name="text">
           <string>快进</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="next_button">
          <property name="text">
           <string>下一张壁纸</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="directory_groupbox">
     <property name="title">
      <string>目录设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QMenuListWidget" name="listWidget">
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="accessibleDescription">
         <string/>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
        <item>
         <property name="text">
          <string>dsaddad</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/file.png</normalon>
          </iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sadsadadsa</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/directory.png</normalon>
          </iconset>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="clean_file_button">
          <property name="text">
           <string>清空文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="remove_file_button">
          <property name="text">
           <string>移除文件</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_file_button">
          <property name="text">
           <string>添加文件</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QMenuListWidget</class>
   <extends>QListWidget</extends>
   <header>WallpaperSystem/Controls/QMenuListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
